marketplace/token.go

```diff
@@ -0,0 +1,37 @@
+package marketplace
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
```

Comment:
Should have caught this earlier - but we should be using the `logging` package instead of the std logger - that way it'll get thrown at cloudwatch and formatted as json.

Reply:
Yeah this is a mistake related to the previous one. I've removed this from here. I guess that we'll use `logging` extensively on the middleware or on the handlers to return user friendly messages, and log the error messages.